app/uk/gov/hmrc/eoricommoncomponent/frontend/services/Save4LaterService.scala

```diff
@@ -54,6 +54,11 @@ class Save4LaterService @Inject() (save4LaterConnector: Save4LaterConnector) {
     save4LaterConnector.put[EmailStatus](internalId.id, emailKey, emailStatus)
   }
 
+  def deleteEmail(internalId: InternalId)(implicit hc: HeaderCarrier): Future[Unit] = {
+    logger.info("deleting email address on mongo")
```

Comment:
Should this be info? we can keep this log as debug. The same comment for rest of the logs

Reply:
Yes, I wondered about those log statements.   I'll change them